tests/pr_test.go

```diff
@@ -55,6 +67,12 @@ func TestRunFsCloudExample(t *testing.T) {
 
 	options := setupOptions(t, "cis-fs", domainName, fscloudExampleTerraformDir)
 
+	options.IgnoreUpdates = testhelper.Exemptions{
+		List: []string{
+			"module.cis_instance.module.waf.ibm_cis_ruleset_entrypoint_version.waf_config[0]",
```

Comment:
Can you leave a code comment for each of these ignores pointing to the provider issue please

Reply:
added